sdk/keyvault/keyvault-keys/test/internal/challengeBasedAuthenticationPolicy.spec.ts

```diff
@@ -9,7 +9,7 @@ import {
   AuthenticationChallengeCache,
   AuthenticationChallenge,
   parseWWWAuthenticate
-} from "../../../keyvault-common/src";
+} from "@azure/core-http";
```

Comment:
seems like we have copy-pasted this test file around.... doesn't make sense to test the policy in each client, we only need to test the client, right?

Reply:
I had three copies of the test before keyvault-common. keyvault-common didn't have support for centralized tests, I didn't put the effort there, but only in reusing the code first. Now it's safe to say that these files can be removed in favor of a single test file in core-http, yep.

Reply:
Sounds like this will take some mocking work. I'll save that for a later PR.